src/components/popups/CreateTeam.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import "../../styles/popups/CreateTeam.scss";
+import PropTypes from "prop-types";
+import { Button } from "components/ui/Button";
+import { useNavigate } from "react-router-dom";
+
+const FormField = (props) => {
+  return (
+    <div className="createTeam field">
+      <label className="createTeam label">{props.label}</label>
+      <input
+        className="createTeam input"
+        placeholder="enter here.."
+        value={props.value}
+        onChange={(e) => props.onChange(e.target.value)}
+      />
+    </div>
+  );
+};
+
+FormField.propTypes = {
+  label: PropTypes.string,
+  value: PropTypes.string,
+  onChange: PropTypes.func,
+  type: PropTypes.string,
+};
+
+const CreateTeam = ({ isOpen, onClose, onCreateTeamClick }) => {
+  if (!isOpen) return null;
+  const [teamName, setTeamName] = useState<string>(null);
+
+  const createTeam = () => {
+    //team ID is received here from backend API API API
+    const teamid = "16";
```

Comment:
add todo comment so that it gets highlighted by the IDE and is in the todo list of the IDE to be checked.
```js
// todo ...
```

Reply:
Done in tde team dashboard branch!